src/DssSpell.t.sol

```diff
@@ -2476,4 +2276,23 @@ contract DssSpellTest is DSTest, DSMath {
         }
         assertEq(expectedHash, actualHash);
     }
+
+    function test_RWA002_values() public {
+        vote(address(spell));
+        spell.schedule();
+        hevm.warp(spell.nextCastTime());
+        spell.cast();
+        assertTrue(spell.done());
+
+        bytes32 ilk = bytes32("RWA002-A");
+
+        // Confirm pip value.
+        DSValueAbstract pip = DSValueAbstract(addr.addr("PIP_RWA002"));
+        assertEq(uint256(pip.read()), 22_495_725 * WAD);
+
+        // Confirm Vat.ilk.spot value.
+        (,, uint256 spot,,) = vat.ilks(ilk);
+        assertEq(spot, 21_424_500 * RAY);
```

Comment:
how was this calculated?

Reply:
There's a comment in the spell.

Reply:
Spot price is the pip val divided by the ilk mat.

Here, `22_495_725 / 1.05`

See https://github.com/makerdao/dss/blob/master/src/spot.sol#L100

Reply:
You can see the conversion from oracle price to `spot` here:
https://github.com/makerdao/dss/blob/master/src/spot.sol#L100

Reply:
(well I got it)

Reply:
Add logic to test drawing DAI as close as possible to the debt ceiling.